resources/zib/zib-TNMTumorClassification.RegionalLymphNodesLocalization.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
```

Comment:
The comment made in the DistantMetastasisLocalization profile applies here as well.